specs/commandline/certificate.md

```diff
@@ -0,0 +1,228 @@
+# notation certificate
+
+## Description
+
+Use ```notation certificate``` command to add/list/delete certificates in notation's trust store. Updating an existing certificate is not allowed since the thumbprint will be inconsistent, which results in a new certificate.
+
+The trust store is in the format of a directory in the filesystem as`x509/<type>/<name>/*.crt|*.cer|*.pem`. Currently two types of trust store are supported:
+
+* `Certificate Authority`: The directory name is `ca`.
+* `Signing Authority`: The directory name is `signingAuthority`
+
+There could be more trust store types introduced in the future.
+
+Here is an example of trust store directory structure:
+
+```text
+$XDG_CONFIG_HOME/notation/truststore
+    /x509
+        /ca
+            /acme-rockets
+                cert1.pem
+                cert2.pem
+                  /sub-dir       # sub directory is ignored
+                    cert-3.pem   # certs under sub directory is ignored
+
+        /signingAuthority
+            /wabbit-networks
+                cert3.crt
+```
+
+In this example, there are two certificates stored in trust store named `acme-rockets` of type `ca`. There is one certificate stored in trust store named `wabbit-networks` of type `signingAuthority`.
+
+Please be noted there will be user level trust store and system level trust store. See [Directory spec](https://github.com/notaryproject/notation/blob/main/specs/directory.md) for more details. The commands `notation certificate add` and `notation certificate delete` are performed only on user level trust store.
+
+## Outline
+
+### notation certificate
+
+```text
+Manage certificates in trust store for signature verification
+
+Usage:
+  notation certificate [command]
+
+Aliases:
+  certificate, cert
+
+Available Commands:
+  add           Add certificates to the trust store. This command only operates on User level.
+  delete        Delete certificates from the trust store. This command only operates on User level.
+  generate-test Generate a test RSA key and a corresponding self-signed certificate.
+  list          List certificates used for verification. This command operates on User level and System level.
```

Comment:
```suggestion
  list          List certificates in the trust store. This command operates on both User level and System level.
```
Programmatically, this line has to align with the short description shown by 'notation certificate list -h'.